### ISSUE TITLE
No image is saving after calling Croppa::url() from view

### ISSUE THREAD

**Issue by shahjahanrasel** [BKWLD/croppa#104]
I have having a strange problem.. after calling Croppa::url() i should get the resized image to be saved in the folder but its not happening.

can you please tell me the reason?

**Comment by weotch** [comment]
Post the contents of your config file, the absolute paths to your src and crops dirs, and the argument signature of your call to `Croppa::url()`

**Comment by ArtbyArjan** [comment]
We've been forced to downgrade as well, because 4.2.1 wasn't saving any cropped images.
We're using croppa inside Laravel 5.0 and have created a method for saving and cropping images.
Images are saved in the public/uploads folder and we create a unique path with the following code:
```php   
$subdir         = strrev( strval( time() ) );
$img_path       = substr( $subdir, 0, 2 ) . '/' . substr( $subdir, 2 ,2 );
$path           = base_path() . '/public/uploads/' . $img_path;
$uniq_name      = hash_file('sha256', $file ) . '.' . $file->getClientOriginalExtension();
```

Cropped images are created by calling a method in the model, like this:
```php
public function getImagesmallAttribute()
    {
        if( $this->image_path ){
            $path_exploded = explode('.', $this->image_path);
            $ret = $path_exploded[0] . '-640x390.' . $path_exploded[1];
            return $ret;
        }
    }
```
So they're only generated when this function is being called in a view, like this:
```php
$model->imagesmall
```
I hope this is enough information for you?
With version 4.2.1 we had the problem that no cropped versions were generated. The original images was saved in the public/uploads folder.

**Comment by weotch** [comment]
@ArtbyArjan did you set the `signing_key` config to `false`? https://github.com/BKWLD/croppa/blob/4.2.1/src/config/config.php#L78-L90

**Comment by ArtbyArjan** [comment]
Sorry for the late reply.

The `singing_key` config isn't set at all. Is that a problem?

**Comment by weotch** [comment]
Not for 4.0.3 ... but I believe it would be the source of your error in 4.2.x

**Comment by ArtbyArjan** [comment]
Thanks for your reply.
I'll keep this in mind for the future, and test it in a new project.

**Comment by nicolasbinet** [comment]
@ArtbyArjan I've just had the same error with 4.2.x, and finally figured out that nginx was responsible for the problem. The request never made it's way to Croppa.

I had setup "alias' folder in nginx to point to my storage/uploads & storage/cache folders.
Now, I've made symbolic links to those two folders and it works perfectly.

I hope this helps you.

**Comment by ArtbyArjan** [comment]
@nicolasbinet Thanks for your reply. When we decide to update to a newer version, I'll be sure to check that out.

**Comment by sindey1982** [comment]
@nicolasbinet I've also foundd the nginx request problem. Handler->handler(in src folder) dosen't excute when route partern match. However works properly in apahce(wamp) . could you show your niginx config ? i'am not familiar with nginx,and tries had all failed.  THKS

**Comment by ArtbyArjan** [comment]
It's been a while, but we were getting errors in a live project.
Cropped images weren't loaded, so I came here again and followed @weotch 's suggestion about setting the `signing_key` to `false` (we're using 4.2.1 for that project)

Problem solved!

**Comment by weotch** [comment]
Some nginx docs were added in #118, maybe they'll help ya

**Comment by Diwas777** [comment]
I want to reduce vertical and horizontal resolution of cropped image.
I have set up as below:
:width="200"
:height="200"
:quality="1"

And while uploading I got image of size (width= 200px and height=200px) which is Ok. But I also want to reduce image horizontal resolution and vertical resolution .How can I do this with Croppa?

 My Case Horizontal resolution and Vertical Resolution is always 96dpi.
![capture](https://user-images.githubusercontent.com/28994482/44074102-00dbdfda-9fb7-11e8-8c39-3abf0181d4a2.PNG)


Also my cropped image size is always greater than original image?
